app/serializers/provider_serializer.rb

```diff
@@ -15,9 +15,12 @@ def name
   end
 
   def address
-    { street: object.AddressStreet, city: object.AddressCity,
-      state: object.AddressStateProvince,
-      zip: object.AddressPostalCode }
+    if object.AddressCity
```

Comment:
is AddressCity the deciding piece of info or do you need to check for the others as well?